zeebe/exporter-common/src/main/java/io/camunda/zeebe/exporter/common/auditlog/transformers/AuditLogTransformerConfigs.java

```diff
@@ -36,4 +38,11 @@ public class AuditLogTransformerConfigs {
               BatchOperationIntent.SUSPEND,
               BatchOperationIntent.CANCEL),
           Set.of(RejectionType.INVALID_STATE));
+
+  public static final TransformerConfig DECISION_EVALUATION_CONFIG =
+      new TransformerConfig(
+          DECISION_EVALUATION,
+          Set.of(DecisionEvaluationIntent.EVALUATED),
+          Set.of(DecisionEvaluationIntent.FAILED),
```

Comment:
🤔 I need to check how the `DecisionEvaluationIntent.FAILED` is created. I think this is actually a Zeebe event, and not a rejection.

If we configure it as a rejection audit log, the data will not be correctly mapped.